packages/unit/test/widget/pageable_list_test.dart

```diff
@@ -79,4 +98,28 @@ void main() {
       expect(currentPage, equals(5));
     });
   });
+
+  test('PageableList resize parent', () {
+    testWidgets((WidgetTester tester) {
+      tester.pumpWidget(new Container());
+      currentPage = null;
+      itemsWrap = true;
+
+      tester.pumpWidget(buildFrame());
+      expect(currentPage, isNull);
+      pageRight(tester);
+      expect(currentPage, equals(5));
+
+      pageSize = new Size(pageSize.height, pageSize.width);
+
+      tester.pumpWidget(buildFrame());
+
```

Comment:
Similarly I think it would be worth checking item five's again size here (it should be 600x800). Even though the tests below imply that only item 5 is visible, ensuring that it has the right size would seal the deal.